lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,56 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleBundleDefinitionSection specifies the embedding mode ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ * <p>
+ * <b>Module bundle section modes</b><br>
+ * <ul>
+ * 	<li>
+ * 		<code>provided</code>: A section of type 'provided' defines a set of modules that should not be included in the bundle file itself, but
```

Comment:
Here it's still `type`. Was that intended?

Reply:
Done